src/meta/src/rpc/service/scale_service.rs

```diff
@@ -92,9 +106,53 @@ where
             .list_worker_node(WorkerType::ComputeNode, None)
             .await;
 
+        let actor_splits = self
+            .source_manager
+            .get_actor_splits()
+            .await
+            .into_iter()
+            .map(|(id, split_impl_vec)| {
+                let multiple_splits = MultipleSplits {
+                    // Currently orchestrator only support Kafka Source
+                    items: split_impl_vec
+                        .into_iter()
+                        .filter(|split_impl| split_impl.as_kafka().is_some())
+                        .map(|split_impl| {
+                            let (topic, partition) =
+                                split_impl.as_kafka().unwrap().get_topic_and_partition();
```

Comment:
```suggestion
                        .flat_map(|split_impl| {
                            let (topic, partition) = split_impl.as_kafka()?.get_topic_and_partition();
```

Reply:
It has been filtered, so I think we can use unwrap?

Reply:
With `flat_map` we can check and unwrap in one go. :)

Reply:
No this problem after using ConnectorSplit